Dockerfile

```diff
@@ -34,7 +34,8 @@ RUN apt-get update && \
         swig \
         xmlto \
         zlib1g-dev && \
-    apt-get clean && \
+        gcc-multilib 
```

Comment:
huh. well spotted. I need to run my docker container with [this 64b->32b trick](https://github.com/docker/docker/issues/611) and then try all this again.

```
root@f4b7d15c7fe6:/spksrc/native/python# uname -a
Linux f4b7d15c7fe6 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt11-1+deb8u6 (2015-11-09) x86_64 GNU/Linux
```